Quran/Container.swift

```diff
@@ -110,7 +110,10 @@ class Container {
     }
 
     func createSettingsController() -> UIViewController {
-        return SettingsNavigationController(rootViewController: SettingsViewController(creators: createSettingsCreators()))
+        var settingsCreators = createSettingsCreators()
+        let settingsController = SettingsViewController(creators: settingsCreators)
+        settingsCreators.parentController = settingsController
```

Comment:
I have to add this because in order to dismiss `MFMailComposeViewController` its delegate method, `mailComposeController(_ controller: MFMailComposeViewController)` needed to be implement. So I added is to Settings controller and gave its reference to creators, to set it when showing.

Reply:
👍